src/utils/Processing.ts

```diff
@@ -220,16 +228,41 @@ export function loadAutomaticDependencies(
   defs: FHIRDefinitions
 ): Promise<void | FHIRDefinitions>[] {
   return AUTOMATIC_DEPENDENCIES.map(dep => {
-    if (configuredDependencies.some(cd => cd.packageId === dep.packageId)) {
+    const alreadyConfigured = configuredDependencies.some(cd => {
+      // hl7.some.package, hl7.some.package.r4, and hl7.somepackage.r5 all represent the same content,
+      // so they are essentially interchangable and we should allow for any of them in the config.
+      // See: https://chat.fhir.org/#narrow/stream/179239-tooling/topic/New.20Implicit.20Package/near/325488084
+      const [configRootId, packageRootId] = [cd.packageId, dep.packageId].map(id =>
+        /\.r[4-9]$/.test(id) ? id.slice(0, -3) : id
+      );
+      return configRootId === packageRootId;
+    });
+    if (alreadyConfigured) {
       return Promise.resolve();
     } else {
-      return loadDependency(dep.packageId, dep.version, defs).catch(e => {
-        let message = `Failed to load automatically-provided ${dep.packageId}#${dep.version}: ${e.message}`;
-        if (/certificate/.test(e.message)) {
-          message += CERTIFICATE_MESSAGE;
-        }
-        logger.warn(message);
-      });
+      let p = Promise.resolve();
+      if (dep.version === 'latest') {
```

Comment:
This doesn't effect this PR since this is going into `master`, but is this the first time we support `'latest'` versions? If so, is this something FPL should eventually handle?

Reply:
It is. I don't think that `latest` as a version identifier is an official FHIR thing -- but it seemed like the right approach to me, given the requirements. As for supporting it in FPL... maybe? This logic is currently outside of FPL since it's part of determining the set of packages we'd pass into FPL.  But it might be nice to embed it in FPL as a feature.

Reply:
That makes sense. I wonder if it could be a helper function that is available from FPL. Thoughts for later!